### DISCUSSION TITLE
EyeTrack alternative cameras and schematics

### DISCUSSION THREAD

**Discussion by nikolaskaralis** [sn-lab/MouseGoggles#29] [Q&A]
Have you tried by any chance the alternative camera you suggest ([IMX519](https://www.arducam.com/product/arducam-mini-16mp-imx519-camera-module-for-raspberry-pi-zero-b0391/)) with Raspberry Pi Zero 2W?

Also, do you happen to have any schematics or photos of the EyeTrack 1.0 version or the Duo 1.5 version to help understand how it looks compared to the older version?

Thanks in advance!

**Comment by misaacson01** [comment]
@nikolaskaralis Yes I have tried that camera, and it does work with the Pi Zero 2W. I'm currently working on an updated version of the EyeTrack headset to use this camera along with the Pi Zero W, since the 2W is often out of stock. The new version I'm working on requires a slight adjustment to the camera slot on the eyepiece to fit the slightly larger camera, and may have a larger headset backing so that the Pi Zeros can stack on the back of the headset to reduce the amount of wires coming off the headset (EyeTrack 1.0 is a bit unwieldy that way). 

EyeTrack 1.0 is included in our revised manuscript which includes a render of the headset, which should be published online soon. I've attached a couple pictures of our first prototype of the EyeTrack 1.0 headset, which is definitely more bulky compared to the earlier versions, partly because of the larger wire harnesses used by the displays from Duo 1.5, but especially due to the cameras, LEDs, and extra Raspberry Pis for eye-tracking. The new EyeTrack version I'm working on right now should be much cleaner again.

![EyeTrack 1 0 Prototype 1](https://github.com/user-attachments/assets/e9db9fe8-61a7-4112-92ac-ba9f3032a3e3)
![EyeTrack 1 0 Prototype 2](https://github.com/user-attachments/assets/5400e452-2e69-4b3d-ae78-2b8a01f646c1)
